Kernel/TTY/SlavePTY.h

```diff
@@ -16,6 +16,7 @@ class SlavePTY final : public TTY {
     // ^TTY
     virtual StringView tty_name() const override;
     virtual ssize_t on_tty_write(const u8*, ssize_t) override;
+    virtual void echo(u8);
```

Comment:
Please mark this `override`:
```suggestion
    virtual void echo(u8) override;
```